src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -3027,7 +3042,27 @@ static void Main()
     }
 }
 ";
-            var comp = CompileAndVerify(source, expectedOutput: " (1, 2)", additionalRefs: new[] { ValueTupleRef, SystemRuntimeFacadeRef });
+
+            Action<ModuleSymbol> validator = (ModuleSymbol module) =>
+            {
+                var sourceModule = (SourceModuleSymbol)module;
+                var compilation = sourceModule.DeclaringCompilation;
+                var tree = compilation.SyntaxTrees.First();
+                var model = compilation.GetSemanticModel(tree);
+
+                var literal = tree.GetRoot().DescendantNodes().OfType<TupleExpressionSyntax>().First();
+                Assert.Equal(@"(null, (1, 2))", literal.ToString());
+                Assert.Null(model.GetTypeInfo(literal).Type);
+                Assert.Equal("(System.String, (System.Int32, System.Int32))", model.GetTypeInfo(literal).ConvertedType.ToTestDisplayString());
+                Assert.Equal(ConversionKind.ImplicitTupleLiteral, model.GetConversion(literal).Kind);
+
+                var nestedLiteral = literal.Arguments[1];
+                Assert.Equal(@"(1, 2)", nestedLiteral.ToString());
+                Assert.Null(model.GetTypeInfo(nestedLiteral).Type);
+                Assert.Null(model.GetTypeInfo(nestedLiteral).ConvertedType);
```

Comment:
I think `(1,2)` must have a type and a converted type. You should be able to extract it into a temp/method and without a type it would be impossible. 
I am actually surprised this does not "just work". It is a trivial tuple literal, it should have at least the natural type... 
The problem could be somewhere else.

It would be acceptable to merge this as is and enter an issue to follow up.